### PR TITLE
Update people group documentation

### PR DESCRIPTION
**Pull request by choldgraf**
This is a mostly cosmetic update to our job titles section in anticipation of adding a few more as we refine the project manager and team manager roles, and add an administrative assistant role.

It renames `Human Resources` to `People Group` (and `hr/` to `people/`), as it seems many companies are moving away from called people "resources", and since most of this work is about bringing in, promoting, paying, etc _people_, it feels reasonable to call it this.

### CODE REVIEW

File: people/titles/engineer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Open Source Infrastructure Engineer
+
+An Open Source Infrastructure (OSIE) focuses on infrastructure that supports interactive computing.
+It intersects job titles such as “dev-ops engineer”, “site reliability engineer”, “software engineer”, and “cloud engineer”.
+
+## Key responsibilities
+
+- Ensure the reliable operation of the 2i2c infrastructure (leveraging production-ready cloud-based tools such as JupyterHub, BinderHub and Dask).
+- Attend to support-facing tickets
```
Comment (sgibson91) [inline people/titles/engineer.md:9]: Something is nagging at me regarding "support-facing" being internal vernacular we use, but might not be very clear to someone reading this in order to info gather before making an application, for example. Can't quite express this nag though.

File: people/titles/engineer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Experience with major cloud providers.
+- Experience in programming and software engineering with a track record of leadership in open, collaborative projects with broad community adoption.
+- Experience working on geographically distributed open-source projects.
+- Experience with the Jupyter ecosystem and other tools for interactive computing.
+- Evidence of existing connections and relationships in the worldwide ecosystem of open source software for data-intensive research and ability to establish new ones.
+- Experience with common data science methods, platforms, workflows, and infrastructures; with data management systems, practices, and standards; and the capacity to gain familiarity with new related topics.
+- Experience engaging with highly technical researchers across a variety of methodological fields, research domains, and computational platforms.
+- Experience building and maintaining continuous deployment pipelines.
+- Interpersonal skills to work with researchers and students. Include having skills to communicate complex information in a clear and concise manner both verbally and in writing
```
Comment (sgibson91) [inline people/titles/engineer.md:33]:
```suggestion
- Interpersonal skills to work with researchers and students. Including the skills to communicate complex information in a clear and concise manner both verbally and in writing
```

File: people/titles/engineer.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Open Source Infrastructure Engineer
+
+An Open Source Infrastructure (OSIE) focuses on infrastructure that supports interactive computing.
+It intersects job titles such as “dev-ops engineer”, “site reliability engineer”, “software engineer”, and “cloud engineer”.
+
+## Key responsibilities
```
Comment (damianavila) [inline people/titles/engineer.md:6]: All the content on this page resembles a lot of the job posts we publish which, I think, makes sense.

**Review by damianavila — APPROVED** [review]
LGTM once @sgibson91 comments are addresed.

File: people/titles/engineer.md (hunk 3 above)
Comment (choldgraf) [reply people/titles/engineer.md:6]: That is one of the goals - we should be able to re-use the content here relatively quickly for job posts.

File: people/titles/engineer.md (hunk 1 above)
Comment (choldgraf) [inline people/titles/engineer.md:9]:
```suggestion
- Attend to operational issues that are surfaced by our support team
```

Comment (choldgraf) [reply people/titles/engineer.md:9]: how's that @sgibson91 ? a bit clearer?

**Review by choldgraf — COMMENTED** [review]
OK I think I've addressed the comments - @damianavila would you like to update the job posting with updated language that @sgibson91 suggested?

File: people/titles/engineer.md (hunk 1 above)
Comment (sgibson91) [reply people/titles/engineer.md:9]: Yep!

**Comment by choldgraf** [comment]
Awesome thanks - OK I'm gonna merge this one in so that we don't create more merge conflicts w/ the folder rename

**Comment by damianavila** [comment]
>  @damianavila would you like to update the job posting with updated language that @sgibson91 suggested?

Adding it to my todo list.

**Pull request merged**